tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "gts/tsconfig-google.json",
   "compilerOptions": {
-    "outDir": "./types/",
+    "outDir": "build/types/",
```

Comment:
We weren't seeing errors because the types were written to the wrong directory, and not being copied into the resulting packages :-/

This will allow us to see the ripple errors again

Reply:
@moog16 I can split this (critical) change out into a different PR if you want. This PR should go in as-is without the ripple fix since it does fix all type errors except for it. The next PR will fix the ripple issue (I traced the issue to the webpack code).